app/backend/src/couchers/urls.py

```diff
@@ -1,6 +1,18 @@
+"""
+Please make sure this file stays in sync with
+//app/frontend/src/routes.ts
+"""
```

Comment:
No, //docs/urls.md is the primary source of URL information that both frontend and backend should refer to.

Please describe it with a comment instead of a docstring as it only makes sense to developers reading the source, not programmers who want to use the functions therein.